test/Connection.js

```diff
@@ -80,13 +88,87 @@ describe('Connection', () => {
     })
   }
 
+  const handlesErrors = (method, requestPath, sendRequest) => {
+    const testCases = [
+      {
+        title: 'should handle 403 reponse',
```

Comment:
This `reponse` means `response`? other test case titles include this 
similarly :)

Reply:
Yes, it should have been `response` :man_facepalming: . I'll fix the typo in a moment.